metadata-ingestion/src/datahub/entrypoints.py

```diff
@@ -77,10 +78,27 @@ def ingest(config: str):
 
 @datahub.command(context_settings=DEFAULT_CONTEXT_SETTINGS)
 def ingest_list_plugins():
+    """List enabled ingestion plugins"""
+
     click.secho("Sources:", bold=True)
     click.echo(str(source_registry))
     click.echo()
     click.secho("Sinks:", bold=True)
     click.echo(str(sink_registry))
     click.echo()
     click.echo('If a plugin is disabled, try running: pip install ".[<plugin>]"')
+
+
+@datahub.command(context_settings=DEFAULT_CONTEXT_SETTINGS)
+@click.option(
+    "-f",
+    "--json-file",
+    type=click.Path(exists=True, dir_okay=False),
+    help="JSON file with a list of MCE objects",
+    required=True,
+)
+def check_mce_schema(json_file: str):
```

Comment:
It seems very specific to add a check_mce_schema command to the top-level `datahub` cli. 

Could we achieve the same outcome (validate that inputs look good) by adding a --dry-run mode to ingest ? 

So `datahub ingest -c mce_to_console.yml --dry-run`

Reply:
Yep agreed that it probably doesn't deserve a top-level command

What should `--dry-run` do exactly - perhaps it should just set the sink to `console`?